src/stories/Checkbox/Checkbox.css

```diff
@@ -0,0 +1,84 @@
+body{
+    font-family:Arial, Helvetica, sans-serif;
+    font-size: 18px;
```

Comment:
Could I please request that we use `rem` instead of `px` wherever possible? 

To make things easier, please add this to our CSS:

```
html {
    font-size: 62.5%; /* 62.5% of 16px = 10px */
}
```

And then please refactor all `px` units to `rem` units for this entire task